tests/ert/unit_tests/dark_storage/test_dark_storage_state.py

```diff
@@ -0,0 +1,115 @@
+import gc
+import io
+import os
+from urllib.parse import quote
+from uuid import UUID
+
+import hypothesis.strategies as st
+import pandas as pd
+import polars
+import pytest
+from hypothesis import assume
+from hypothesis.stateful import rule
+from starlette.testclient import TestClient
+
+from ert.dark_storage import enkf
+from ert.dark_storage.app import app
+from tests.ert.unit_tests.storage.test_local_storage import StatefulStorageTest
+
+
+def escape(s):
+    return quote(quote(quote(s, safe="")))
+
+
+class DarkStorageStateTest(StatefulStorageTest):
+    def __init__(self):
+        super().__init__()
+        self.prev_no_token = os.environ.get("ERT_STORAGE_NO_TOKEN")
+        self.prev_ens_path = os.environ.get("ERT_STORAGE_ENS_PATH")
+        os.environ["ERT_STORAGE_NO_TOKEN"] = "yup"
+        os.environ["ERT_STORAGE_ENS_PATH"] = str(self.storage.path)
+        self.client = TestClient(app)
+
+    @rule()
+    def get_experiments_through_client(self):
+        self.client.get("/updates/storage")
+        response = self.client.get("/experiments")
+        experiment_records = response.json()
+        assert len(experiment_records) == len(list(self.storage.experiments))
```

Comment:
Can we check that this is not zero also?

Reply:
I think it could be empty if we haven't initialized anything in storage yet.